popup.js

```diff
@@ -0,0 +1,175 @@
+                        <li>Backend Dev</li>
+                        <li><img src="assets/images/Counter.png" alt="dot"></li>
+                        <li>2015</li>
+                    </ul>
+                    </div>
+                    <p class="para">`+cardsObj[i].description+`</p> 
+                        <div class="button-list">
+                            <ul>
+                              <li><a href="">html</a></li>
+                              <li><a href="">css</a></li>
+                              <li><a href="">javaScript</a></li>
+                            </ul>
+                          </div>
+                      <div> 
+                    <button class="btn pop-up" data-id="" id="popup-btn"  type="button">See Project</button>
+                    </div>
+                </div>
+            </div>`;
+        //    i++;        
+
+              // <i class="btnicon bi bi-github"></i>
+        
+    }
+
+       
+    
+function Popup(position) {
+    const popContent = document.createElement('section');
+    popContent.innerHTML = (`
+    <div class="pop-content">
+    <div class="head-title">
+      <h3>${cardsObj[position].title}</h3>
+      <span class="close-icon">&times;</span>
+    </div>
+    <ul class="work-place">
+      <li class="year">${
+        cardsObj[position].jobDescription.company
+      }&nbsp;&nbsp; &#8226; &nbsp;</li>
+      <li class="year gray">${
+        cardsObj[position].jobDescription.role
+      }&nbsp;&nbsp; &#8226; &nbsp;</li>
+      <li class="year gray">${cardsObj[position].jobDescription.year}</li>
+    </ul>
+    <img class= "popimg" src="${cardsObj[position].mimg}" alt="">
+    <img class= "dpopimg" src="${cardsObj[position].dimg}" alt="">
+    <div class="bottom">
+      <p>${cardsObj[position].description}</p>
+      <div class="right-side">
+        <ul class="project-languages">
+          <li ><a class= "programming" href="">${cardsObj[position].skills[0]}</a></li>
+          <li ><a class= "programming" href="">${cardsObj[position].skills[1]}</a></li>
+          <li ><a class= "programming" href="">${cardsObj[position].skills[2]}</a></li>
+        </ul>
+        <div class="popup-buttons">
+          <button type="button" class="pop-btn">
+                  <a class="btn-live"href="https://abhishekkm05.github.io/Mobile-Portfolio/">
+                  ${
+                    cardsObj[position].btn1
+      } &nbsp;&nbsp;&nbsp;
+      <img src="./assets/images/source.svg">
+
+  
+                  </a>
+          </button>
+          <button type="button" class="pop-btn">
+                  <a class="btn-live"href="https://github.com/AbhishekkM05/Mobile-Portfolio/tree/pop-up">
+                  ${
+                    cardsObj[position].btn2
+      } &nbsp;&nbsp;&nbsp;         
+      <img src="./assets/images/live.svg">
+
+    
+     
+      
+                  </a>
+          </button>
+      </div>
+    </div>
+    </div>`);
```

Comment:
- [ ] Your popup window is working as expected, but the style of the popup has some issues. The mobile version of the popup does not cover the entire screen when the user clicks on it, and the popup is not displayed on the center of the screen so the user needs to scroll up a little bit to see the popup information and the 'x' button. It also leaves the bottom without the blur overlay effect. Please make the popup fill the whole screen and look like it is floating above the content of the page.
 -Your mobile popup:
<img width="480" alt="image" src="https://user-images.githubusercontent.com/61919617/197178563-1d3f163d-e757-427f-8976-388ff6317acc.png">

 - [ ] On the desktop version the popup does not have top and bottom margins like the template and is not responsive, so depending on the users' screen size, the popup can be half cut out of the screen making a bad user experience. Please make the popup size responsive so every user can have a good experience using it.
 - Your desktop version:
 - 
![Screen Recording 2022-10-21 at 7 48 12 AM](https://user-images.githubusercontent.com/61919617/197179620-06e20abe-1ad9-4c40-8fa6-501f04da26b3.gif)

 - [ ] The project information list items, project description, project languages, and buttons do not have the style used in the Figma template. Kindly add the style so your popup window can look more professional and have a consistent style with the project cards.